README.md

```diff
@@ -2,45 +2,100 @@
 
 [![GoDoc](https://pkg.go.dev/badge/github.com/brave-experiments/nitriding?utm_source=godoc)](https://pkg.go.dev/github.com/brave-experiments/nitriding)
 
-This package helps with building networked Go applications on top of AWS Nitro
+This package helps with building Go-based Web applications on top of AWS Nitro
 Enclaves.  The package provides the following features:
 
-1. Initialize the enclave's entropy pool.
+1. Automatically obtains an HTTPS certificate (either self-signed or via [Let's
+   Encrypt](https://letsencrypt.org)) for clients to securely connect to your
+   enclave over the Internet.
 
-2. Obtain an HTTPS certificate for clients to connect to the enclave; either
-   self-signed, or via Let's Encrypt.
+2. Automatically exposes an HTTPS endpoint for remote attestation.  After
+   having audited your enclave's source code, your users can conveniently
+   verify the enclave by using a tool like
+   [verify-enclave](https://github.com/brave-experiments/verify-enclave)
+   and by running:
```

Comment:
s/by// Keep the command line as part of the same adverbial clause modifying "can...verify". The second "by" creates a second, semanticly-independent clause.

Reply:
Fixed in 0bd94ad.